main-app/app/scripts/game/services/bingo-call.js

```diff
@@ -1,29 +1,59 @@
 (function () {
     'use strict';
     angular.module('Tombola.Games.Bingo90.Core')
-        .service('BingoCall',  ['BingoTicket','GameProxy', function (bingoTicket, gameProxy) {
+        .service('BingoCall',
+        ['$interval', '$state', 'BingoTicket', 'GameProxy', 'EndOfGame', 'CoreApiConverter', function ($interval,
+                                                                                                       $state,
+                                                                                                       bingoTicket,
+                                                                                                       gameProxy,
+                                                                                                       endOfGame,
+                                                                                                       coreApiConverter) {
             var me = this,
+                removeFirstElement = function () {
+                    if (me.lastCallsDisplay.length >= 6) {
+                        me.lastCallsDisplay.splice(0, 1);
+                    }
+                },
 
-                getCallData = {
-                    gameId: 1,
-                    userId: "drwho",
-                    balance: 100000000000,
-                    callnumber: 1
+                checkForNumber = function (callNumber) {
+                    if (callNumber.toString().length < 2) {
+                        callNumber = "0" + callNumber.toString();
+                    }
+
+                    if (bingoTicket.balls.indexOf(callNumber.toString()) !== -1) {
+                        bingoTicket.markedNumbers[bingoTicket.balls.indexOf(callNumber.toString())] = "markedCell";
+                    }
                 },
 
-            makeApiCall = function(apiName, action, data, token){
-                gameProxy.callApi(apiName, action, data, token)
-                    .then(function (data) {
-                        me.callMessage = data;
-                        console.log(data);
-                    }).catch(function (data) {
-                        /* Error stub */
-                        console.log(data);
-                    });
-            };
+                makeApiCall = function (apiName, action, data) {
+                    gameProxy.callApi(apiName, action, data)
+                        .then(function (data) {
+                            me.callMessage = data;
+                            me.lastCallsDisplay.push(data.payload.call);
```

Comment:
Create a new function out of this and the next line - this adds a new call to the "recents" list and removes if necessary.